types/types.d.ts

```diff
@@ -2,6 +2,7 @@
 
 import type * as Fastify from 'fastify';
 import { FastifyPluginCallback } from 'fastify';
+import { SerializeOptions } from "@fastify/cookie/types/plugin"
```

Comment:
Please avoid a deep require, all public interfaces should be exported from the entry point.

Reply:
I see, updated the PR